openwisp_monitoring/device/templates/admin/config/device/change_form.html

```diff
@@ -141,6 +141,31 @@ <h2>{% trans 'Interface status' %}: {{ interface.name }}</h2>
                   {% endif %}
                   {% endif %}
               </div>
+              <div class = "device_addresses">
+                {% if interface.addresses %}
+                  <h2>{% trans 'ADDRESSES' %}:</h2>
+                    <table>
+                      <thead>
+                        <tr>
```

Comment:
be consistent with indentation, 4 space indentation is being used in this file, so follow it please. Adopt the same methodology on any file you work on please.